static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -83,13 +90,27 @@ function UserActionsNavigator({event, crumbs}: Props) {
       if (startTimestamp) {
         setCurrentHoverTime(relativeTimeInMs(item.timestamp ?? '', startTimestamp));
       }
+
+      if (item.data && 'nodeId' in item.data) {
+        // XXX: Kind of hacky, but mouseLeave does not fire if you move from a
+        // crumb to a tooltip
+        clearAllHighlights();
```

Comment:
Not sure how to deal with this issue of mouseleave not firing when we move mouse cursor from a breadcrumb to the tooltip, but this will suffice for now.